components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -133,9 +139,34 @@ function CustomLeague:createLiquipediaTierDisplay(args)
 
 	content = content .. '[[Category:' .. tierDisplay .. ' Tournaments]]'
 
+	-- Psyonix icon
+	if CustomLeague:containsPsyonix('organizer') then
+		content = content .. ' ' .. string.format(_PSYONIX_ICON, 'organized')
+	elseif CustomLeague:containsPsyonix('sponsor') then
+		content = content .. ' ' .. string.format(_PSYONIX_ICON, 'sponsored')
+	end
+
 	return content
 end
 
+function CustomLeague:liquipediaTierHighlighted()
```

Comment:
Is this a boolean function? If so:

```suggestion
function CustomLeague:isLiquipediaTierHighlighted()
```

Reply:
The function is inherited from [here](https://github.com/Liquipedia/Lua-Modules/blob/main/components/infobox/commons/infobox_league.lua#L284-L287), so I think it makes sense to keep the same name

Reply:
It does, thanks for pointing that out.